src/Sylius/Behat/Page/Shop/Checkout/AddressPage.php

```diff
@@ -73,10 +67,7 @@ public function chooseDifferentBillingAddress()
         $billingAddressSwitch->check();
     }
 
-    /**
-     * {@inheritdoc}
-     */
-    public function checkInvalidCredentialsValidation()
+    public function checkInvalidCredentialsValidation(): bool
     {
         $this->getElement('login_password')->waitFor(5, function () {
             $validationElement = $this->getElement('login_password')->getParent()->find('css', '.red.label');
```

Comment:
```suggestion
            $validationElement = $this->getElement('login_password')->getParent()->find('css', '.red.label');
```

remove `.red.label` and add data-attributes